controllers/k8ssandra/medusa_reconciler.go

```diff
@@ -83,6 +83,7 @@ func (r *K8ssandraClusterReconciler) reconcileMedusa(
 		// Reconcile the Medusa standalone deployment
 		kcKey := utils.GetKey(kc)
 		desiredMedusaStandalone.SetLabels(labels.CleanedUpByLabels(kcKey))
+		logger.Debug("Desired Medusa standalone deployment", "deployment", desiredMedusaStandalone)
```

Comment:
```suggestion
		logger.Info("Desired Medusa standalone deployment", "deployment", desiredMedusaStandalone)
```

Reply:
```suggestion
```
I'll remove this since desiredMedusaStandalone is very big